src-plugins/itkFilters/itkFiltersToolBox.cpp

```diff
@@ -327,6 +327,11 @@ itkFiltersToolBox::itkFiltersToolBox ( QWidget *parent ) : medAbstractSelectable
     setAboutPluginVisibility ( true );
 
     connect ( runButton, SIGNAL ( clicked() ), this, SLOT ( run() ) );
+
+    if (this->selectorToolBox()) // empty in pipelines
+    {
+        connect(this->selectorToolBox(), SIGNAL(inputChanged()), this, SLOT(update()));
```

Comment:
I would call *update()* here, in case the user drops the image before selecting the filter.